includes/server/class-llms-rest-access-plans-controller.php

```diff
@@ -461,8 +461,13 @@ public function create_item_permissions_check( $request ) {
 			$post_type_object = get_post_type_object( get_post_type( $request['post_id'] ) );
 
 			if ( ! current_user_can( $post_type_object->cap->edit_post, $request['post_id'] ) ) {
-				// Translators: The post type singular name.
-				return llms_rest_authorization_required_error( sprintf( __( 'Sorry, you are not allowed to create a %s as this user.', 'lifterlms' ), $post_type_object->labels->singular_name ) );
+				return llms_rest_authorization_required_error(
+					sprintf(
+						// Translators: The post type singular name.
+						__( 'Sorry, you are not allowed to create a %s as this user.', 'lifterlms' ),
+						$post_type_object->labels->singular_name
```

Comment:
1. I like the `$var = $desc` pattern here, I've never actually seen it used before but I tend towards this myself. 
2. I realize that in normal scenarios (courses and memberships) this won't matter but just in case using the name (plural) without a will prevent potential grammatical issues with post types that start with a vowel.

```suggestion
						// Translators: %s = The post type name.
						__( 'Sorry, you are not allowed to create %s as this user.', 'lifterlms' ),
						$post_type_object->labels->name
```